roles/flannel/tasks/main.yml

```diff
@@ -1,9 +1,11 @@
 ---
 - name: Install nhorman Flannel yum repo file
   copy: src=flannel.repo dest=/etc/yum.repos.d/
+  when: not is_atomic
```

Comment:
Should we drop this entirely?  flannel is in fedora....   do i have any other repos left?